test/uk/gov/hmrc/apprenticeshiplevy/controllers/auth/AuthActionSpec.scala

```diff
@@ -62,19 +69,19 @@ class AuthActionSpec extends AppLevyUnitSpec with GuiceOneAppPerSuite with Retri
       )
     )
 
-  val paRetrieval: Enrolments ~ LegacyCredentials =
-    Enrolments(Set()) ~ PAClientId("app-id")
+  val paRetrieval: Enrolments ~ Option[Credentials] =
```

Comment:
Interesting, haven't seen a type like this before (with the tilde). Anywhere I can read up about that / got a quick explanation?

Reply:
It's part of Auth, see here: https://github.com/hmrc/auth-client/blob/cef945e52d5dc0c8f4a36b12571b90ccccd324ee/src/main/scala/uk/gov/hmrc/auth/core/retrieve/Retrieval.scala#L25